docs/backup.rst

```diff
@@ -0,0 +1,30 @@
+Downloading logs from s3
+========================
+
+1.  Install aws according to
+    `official instruction <https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>`__.
+
+2.  Set up aws
+
+    ..  code-block:: console
+
+        $ aws configure
+
+        AWS Access Key ID: ask teamX
+        AWS Secret Access Key: ask teamX
+        Default region name: ru-msk
+        Default output format: json
+
+3.  Check the bucket content
+
+    ..  code-block:: console
+
+        $  aws s3 --endpoint-url https://hb.bizmrg.com \
+            ls s3://multivac/tarantool/tarantool
+
+4.  Download logs from s3
+
+    ..  code-block:: console
+
+        $ aws s3 --endpoint-url https://hb.bizmrg.com \
+            sync s3://multivac/tarantool/tarantool/workflow_run_jobs/ your_path
```

Comment:
In general, instruction looks great to me.

But don't we want to mention at least approximately the size of the `workflow_run_jobs` directory? The user might not have enough space on his disc to download all the logs. According to NickVolynkin it's something about 50Gb.

Reply:
Actually it's 123Gb now

Reply:
Fixed